presentation/src/main/java/com/kdjj/presentation/di/PresentationModule.kt

```diff
@@ -0,0 +1,26 @@
+package com.kdjj.presentation.di
+
+import android.content.Context
+import android.media.Ringtone
+import android.media.RingtoneManager
+import android.net.Uri
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+class PresentationModule {
+
+    @Provides
+    fun provideRingTone(@ApplicationContext context: Context): Ringtone {
+        val notification1: Uri =
```

Comment:
이 친구는 알림 변수인 것 같은데 혹시 왜 1이 붙어있을까요??
```suggestion
        val notification1: Uri =
```

Reply:
앗 실수로 들어간 것 같습니다 수정할게요